tox.ini

```diff
@@ -10,6 +10,7 @@ deps =
     django111: Django>=1.11,<2
     django20: Django>=2.0,<2.1
     django21: Django>=2.1,<2.2
+    django22: Django>=2.2,<2.3
     -r {toxinidir}/requirements/testing.txt
 commands =
     coverage run ./manage.py test --settings=settings {posargs}
```

Comment:
I think this is suppressing deprecation warnings by default (see the "Test with warnings enabled" box under [Running tests](https://docs.djangoproject.com/en/1.11/topics/testing/overview/#running-tests)).  Probably the easiest way to enable them without switching to pytest is something like `python -Wd -m coverage run ./manage.py ...`.